runtime/compiler/env/J9CPU.cpp

```diff
@@ -43,3 +44,29 @@ J9::CPU::getProcessorDescription()
 #endif /* defined(J9VM_OPT_JITSERVER) */
    return _processorDescription;
    }
+
+void
+J9::CPU::applyFeatureMasks(OMRProcessorDesc& processorDescription, const uint32_t* enabledFeatures, size_t size)
+   {
+   OMRPORT_ACCESS_FROM_OMRPORT(TR::Compiler->omrPortLib);
+   memset(_featureMasks.features, 0, OMRPORT_SYSINFO_FEATURES_SIZE*sizeof(uint32_t));
```

Comment:
Since `_featureMasks` are reset and created in this routine just to be copied over to `processorDescription` why don't we declare the `_featureMasks` as a local in this function?

Reply:
The reason we don't declare `_featureMasks` as a local in this function is because `_featureMasks` will be repeatedly used by `supportsFeature()` to detect query to unused features (which is an illegal action).